docs/modules/ROOT/pages/wk-lab-1-hello-flow-devui.adoc

```diff
@@ -16,18 +16,43 @@ If you don’t have a project yet, use the Quarkus CLI:
 [source,shell]
 ----
 quarkus create app org.acme:hello-flow \
-  --extension=resteasy-reactive-jackson,io.quarkiverse.flow:quarkus-flow
+  --extension=resteasy-reactive-jackson
 cd hello-flow
 ----
 
 Or add RESTEasy Reactive with JSON to an existing project.
 
+[NOTE]
+====
+Quarkus Flow has a Codestart available, which you can use to create a project with a sample workflow and JAX-RS endpoint.
+
+As we are focusing on creating a minimal example here, we won’t use the Codestart in this lab.
+====
+
+== Add the Quarkus Flow extension
+
+Add the Quarkus Flow extension to your project:
+
+
+[source,shell]
+----
+./mvnw quarkus:add-extension -Dextensions="io.quarkus:quarkus-flow"
+----
+
+Or with the Quarkus CLI:
+
+[source,shell]
+----
+quarkus extension add io.quarkus:quarkus-flow
+----
+
 == 2. Create a minimal workflow
 
 Create a `Flow` subclass that returns a simple greeting.
 For example:
 
 [source,java]
+.src/main/java/org/acme/HelloWorkflow.java
```

Comment:
Why are you changing this? The include doesn't make sense anymore?

Reply:
It is true.